network/p2p/p2p_sync.go

```diff
@@ -82,11 +82,11 @@ func (n *p2pNetwork) handleStream(logger *zap.Logger, handler p2pprotocol.Reques
 }
 
 // getSubsetOfPeers returns a subset of the peers from that topic
-func (n *p2pNetwork) getSubsetOfPeers(logger *zap.Logger, senderID []byte, maxPeers int, filter func(peer.ID) bool) (peers []peer.ID, err error) {
+func (n *p2pNetwork) getSubsetOfPeers(logger *zap.Logger, vpk spectypes.ValidatorPK, maxPeers int, filter func(peer.ID) bool) (peers []peer.ID, err error) {
```

Comment:
Why revert back to validator pk, if it could be either validator or committee? (so "sender")

Reply:
https://github.com/ssvlabs/ssv/blob/alan/no-fork-fix-tests/network/p2p/p2p_sync.go#L191

Ill change here to same.